A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSInit.js

```diff
@@ -0,0 +1,128 @@
+class CDUIRSInit {
+    static ShowPage(mcdu, lon) {
+        mcdu.clearDisplay();
+        mcdu.setTitle('IRS INIT');
+        const checkAligned = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Number");
+        const emptyIRSGpsString = "--°--.--/---°--.--"
+        const arrowupdwn = "↑↓"
+        let larrowupdwn = arrowupdwn
+        let rarrowupdwn = ""
+        if (lon) {
+            rarrowupdwn = arrowupdwn
+            larrowupdwn = ""
+        }
+        let statusIRS1;
+        let statusIRS2;
+        let statusIRS3;
+        // Ref coordinates are taken based on origin airport
+        const airportCoordinates = mcdu.flightPlanManager.getOrigin().infos.coordinates;
+        const originAirportLat = CDUInitPage.ConvertDDToDMS(airportCoordinates['lat'], false);
+        const originAirportLon = CDUInitPage.ConvertDDToDMS(airportCoordinates['long'], true);
+        const currentGPSLat = CDUInitPage.ConvertDDToDMS(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"), false);
+        const currentGPSLon = CDUInitPage.ConvertDDToDMS(SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude"), true);
+        let alignMsg = "ALIGN ON REF →[color]blue"
+        let GPSPosAlign = ["--°--.--", "--°--.--", " "]
+        if (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_1", "Enum") || SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_2", "Enum") || SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_3", "Enum")) {
+            GPSPosAlign = ['{IrsInitFont}' + currentGPSLat['deg'] + '°{IrsInitFontEnd}' + currentGPSLat['min'] + '.' + Math.ceil(Number(currentGPSLat['sec'] / 100)) + '[s-text]{IrsInitFont}' + currentGPSLat['dir'] + "{IrsInitFontEnd} [color]green", '{IrsInitFont}' + currentGPSLon['deg'] + '°{IrsInitFontEnd}' + currentGPSLon['min'] + '.' + Math.ceil(Number(currentGPSLon['sec'] / 100)) + '[s-text]{IrsInitFont}' + currentGPSLon['dir'] + "{IrsInitFontEnd} [color]green", ""]
+            if (!SimVar.GetSimVarValue("A32NX_ADIRS_PFD_ALIGNED_FIRST", "boolean") && !SimVar.GetSimVarValue("L:A320_Neo_ADIRS_IN_ALIGN", "Bool") && checkAligned == 1) {
+                alignMsg = "CONFIRM ALIGN* [color]red"
+            }
+        }
+        let IRSAlignOnPos = "{IrsInitFont}" + currentGPSLat['deg'] + '°{IrsInitFontEnd}' + currentGPSLat['min'] + '.' + Math.ceil(Number(currentGPSLat['sec'] / 100)) + '[s-text]{IrsInitFont}' + currentGPSLat['dir'] + '/{IrsInitFontEnd}{IrsInitFont}' + currentGPSLon['deg'] + '°{IrsInitFontEnd}' + currentGPSLon['min'] + '.' + Math.ceil(Number(currentGPSLon['sec'] / 100)) + '{IrsInitFont}' +  currentGPSLon['dir'] + '{IrsInitFontEnd}'
+        let IRS1GpsString = emptyIRSGpsString
+        if (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_1", "Enum") == 1) {
+            if (checkAligned == 2) {
+                statusIRS1 = "IRS1 ALIGNED ON GPS"
+            } else {
+                statusIRS1 = "IRS1 ALIGNING ON GPS"
```

Comment:
**thought**: There's a lot of inconsistency in usage of `;` at the end of lines.
**suggestion**: Consistently add `;` at the end of lines, where applicable.

Reply:
Added missing ; to code.